### PR TITLE
Add high intensity colors

### PR DESCRIPTION
**Pull request by cxfksword**
high intensity colors useful in 256-color mode

### CODE REVIEW

File: ansicolor_windows.go [hunk 1]
```diff
@@ -78,6 +78,16 @@ const (
...
+
+	ansiLightForegroundGray    = "90"
+	ansiLightForegroundRed     = "91"
+	ansiLightForegroundGreen   = "92"
+	ansiLightForegroundYellow  = "93"
+	ansiLightForegroundBlue    = "94"
+	ansiLightForegroundMagenta = "95"
+	ansiLightForegroundCyan    = "96"
+	ansiLightForegroundWhite   = "97"
+	ansiLightForegroundDefault = "99"
```
Comment (shiena) [inline ansicolor_windows.go]: I refer to the following definition, but did not find the definition of 99.
Please you tell me the definition of I to 99.
http://en.wikipedia.org/wiki/ANSI_escape_code#CSI_codes

In addition to that, also I would like to add high-intensity background color, if possible.

**Comment by cxfksword** [comment]
Sorry for my negligence. i will fix and also add high-intensity background color.

**Comment by cxfksword** [comment]
@shiena I have add high-intensity background color.

**Comment by shiena** [comment]
@cxfksword Thank you for commit!
It has been improved thanks.

**Pull request merged**